ailerons_tracker_backend/__init__.py

```diff
@@ -33,74 +34,91 @@ def create_app(test_config=None):
     except OSError:
         pass
 
-    @app.route('/upload', methods=['GET', 'POST'])
+    @app.post('/upload')
     def upload_file():
-        """ Parse a CSV file and insert data in DB
+        """ Parse a CSV file and insert data in DB """
 
-        Returns:
-            204: Successful operation
-            e: Error while attempting insert in DB """
+        try:
+            # A priori on aurait deux fichiers donc j'ai donné un nouveau nom à celui ci "
+            file_name, file_path = csv_parser.prepare_csv(request)
+            csv_id = supabase.create_csv_log(file_name)
+
+            # plus besoin de la table jointe si j'ai bien compris mais je n'y ai pas touché
+            df_list, new_individual_id_list = csv_parser.parse_csv(
+                file_path, csv_id)
+
+            supabase.batch_insert("record", df_list)
+            supabase.batch_insert("individual_record_id",
+                                  new_individual_id_list)
 
-        if request.method == 'POST':
-            try:
-                file = request.files['csvFile']
+            os.remove(file_path)
 
-                if file.filename == '':
-                    raise InvalidFileName()
+            generator = Generator()
+            generator.generate()
```

Comment:
This is the big generator "job", creating GeoJSON data and inserting it in the DB. It's already being reworked extensively but I'll submit another PR for that the diff is already big enough.